src/network/networkd-json.c

```diff
@@ -943,11 +936,11 @@ static int pref64_append_json(Link *link, JsonVariant **v) {
                         return r;
         }
 
-        r = json_append_one(&w, "PREF64", array);
+        r = json_variant_merge_pair(&w, "PREF64", array);
```

Comment:
also, really weird way to say:

```c
r = json_build(&w, JSON_BUILD_OBJECT(JSON_BUILD_PAIR_VARIANT("PREF64", array)));
```

Sure, your line is strictyl speaking shorter, but it's also a lot weirder. Creating objects by merging them with an empty object is just weird.